release-notes/opensearch-dashboards-search-relevance.release-notes-2.4.1.0.md

```diff
@@ -0,0 +1,4 @@
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.4.1
+
+## 🛠 Maintenance
+*  Increment from 2.4 to 2.4.1 ([#104](https://github.com/opensearch-project/dashboards-search-relevance/pull/104))
```

Comment:
nit: should be 2.4.0

Reply:
changed to 2.4.0  in [2.4 changing to 2.4.0](https://github.com/opensearch-project/dashboards-search-relevance/pull/131/commits/52f3814f7a0ec2d0072bf884f271535bbd0fcd99)